server/setup_flow.go

```diff
@@ -268,11 +268,11 @@ func (p *Plugin) stepCloudAddedInstance() flow.Step {
 
 func (p *Plugin) stepCloudEnableDeveloperMode() flow.Step {
 	return flow.NewStep(stepCloudEnableDeveloperMode).
-		WithPretext("##### :white_check_mark: Step 2: configure the Mattermost app in Jira").
+		WithPretext("##### :white_check_mark: Step 2: Configure the Mattermost app in Jira").
```

Comment:
Is this the same step as line 108 and 211?

Reply:
See my comment on line 211, same conceptually, different technically.